cmd/svcat/binding/bind_cmd.go

```diff
@@ -65,10 +76,11 @@ func NewBindCmd(cxt *command.Context) *cobra.Command {
 		"The name of the secret. Defaults to the name of the instance.",
 	)
 	cmd.Flags().StringSliceVarP(&bindCmd.rawParams, "param", "p", nil,
-		"Additional parameter to use when binding the instance, format: NAME=VALUE")
+		"Additional parameter to use when binding the instance, format: NAME=VALUE. Cannot be combined with --params-json")
 	cmd.Flags().StringSliceVarP(&bindCmd.rawSecrets, "secret", "s", nil,
 		"Additional parameter, whose value is stored in a secret, to use when binding the instance, format: SECRET[KEY]")
-
+	cmd.Flags().StringVar(&bindCmd.jsonParams, "params-json", "",
```

Comment:
I like it.

I would like a version of this that was `--secret-params-json` and let you put a json blob into a secret that was referenced.

Reply:
Would the result of that command create a new secret and then use the secret just like we currently do with `--secret-name NAME` ? 

We would have to have some way to clean that secret up after the operation has finished. I think this sounds like a good idea and I am guessing most users would not understand that sending params via svcat exposes those to the other users of k8s.

Reply:
Just to make sure I understand, the difference between `--secret` and this new flag would be that it would handle making the secret for you and setting the secret paramsfrom on the instance for you?

Reply:
@carolynvs I misunderstood the suggestion at first. I think @pmorie was suggesting the simpler case of supporting exactly what the current `--secret` does except provide the secret pairs as a json blob provided by `--secret-json`

Reply:
Oh! @pmorie can you weigh in when you get a chance? 😀